src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -161,6 +172,48 @@ describe('<DateSelector />', () => {
     });
   });
 
+  describe('timezones adjustments', () => {
+    const date = '2019-01-01';
+
+    it('sets the correct selected date with timezone offset -540 on DatePicker and TextField', () => {
+      getTimezoneOffset.mockImplementation(() => -540);
+      const wrapper = mountWithAppProvider(
```

Comment:
Better to have a line break for readability:

- Test setup
- Mount component
- Assert